src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -299,3 +302,80 @@ export const MyHighlightsImage = styled.img`
   height: ${myHighlightsImageHeight}rem;
   margin-top: ${popupBodyPadding}rem;
 `;
+
+// tslint:disable-next-line:variable-name
+export const HighlightsChapter = styled.div`
+  ${h4Style}
+  height: 5.6rem;
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+  background: ${theme.color.neutral.darker};
+  padding: 0 ${popupPadding}rem;
+  width: 100%;
+`;
+
+// tslint:disable-next-line:variable-name
+export const ShowMyHighlightsBody = styled(PopupBody)`
+  padding: 0;
+`;
+
+// tslint:disable-next-line:variable-name
+export const HighlightWrapper = styled.div`
+  margin: 0 1.6rem ${popupPadding}rem;
+  border: solid 0.1rem ${theme.color.neutral.darkest};
+`;
+
+// tslint:disable-next-line:variable-name
+export const HighlightSection = styled.div`
+  ${labelStyle}
+  padding: 0 ${popupBodyPadding}rem 0 ${popupPadding}rem;
+  background: ${theme.color.neutral.darkest};
+  height: 3.2rem;
+  display: flex;
+  align-items: center;
+  font-weight: bold;
+`;
+
+// tslint:disable-next-line:variable-name
+export const HighlightOuterWrapper = styled.div`
+  :not(:last-child) {
+    border-bottom: solid 0.2rem ${theme.color.neutral.darker};
+  }
+`;
+
+// tslint:disable-next-line:variable-name
+export const HighlightContentWrapper = styled.div`
+  padding: 1.2rem ${popupBodyPadding}rem;
+
+  ${highlightStyles.map((style) => css`
+    &.highlight.${style.label} {
+      border-left: solid 0.8rem ${style.focused};
+    }
+  `)}
```

Comment:
this seems like it'd be easier to set the style label as a prop instead of a class and then only render the border for the one style based on that prop instead of putting the css for all of them